pages/create-issue.tsx

```diff
@@ -1,14 +1,16 @@
 import { GetStaticProps } from 'next'
-import React, { useState } from 'react';
+import { title } from 'process';
+import React, { useEffect, useState } from 'react';
+import { toNumber } from '../helpers/number-transformation';
 import BeproService from '../services/bepro';
 import GithubMicroService from '../services/github-microservice';
 
 export default function PageCreateIssue() {
 
-
   const [issueTitle, setIssueTitle] = useState("");
   const [issueDescription, setIssueDescription] = useState("");
-  const [issueAmount, setIssueAmount] = useState<number>(0);
+  const [issueAmount, setIssueAmount] = useState<string>('0');
```

Comment:
`issueAmount` can't be a string because `balance` isn't a string. When comparing the two we will need to make type casting when we shouldn't need to. These 2 values must be of the same _type_

Reply:
here I have a problem. Because status as number does not allow writing "," and "."